### PR TITLE
Changed default serialization format for BigDecimal and DateTime

### PR DESCRIPTION
**Pull request by TheDro**

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -44,7 +44,7 @@ GEM
     rspec-support (3.10.0)
     sqlite3 (1.4.2)
     thread_safe (0.3.6)
-    tzinfo (1.2.8)
+    tzinfo (1.2.9)
```
Comment (jscharf) [inline Gemfile.lock:47]: Did you need the latest version for this change or are you just updating this to stay current?

Comment (TheDro) [reply Gemfile.lock:47]: Just staying current. It's not an intentional change.

File: lib/gql_serializer.rb [hunk 2]
```diff
@@ -98,6 +98,13 @@ def self.serialize(record, hasharray, options)
     hash
   end
 
+  def self.coerce_value(value)
+    return value.to_f if value.is_a? BigDecimal
+    return value.new_offset(0).strftime("%FT%TZ") if value.is_a? DateTime
+    return value.utc.iso8601 if value.is_a? Time
```
Comment (TheDro) [inline lib/gql_serializer.rb:104]: I'm adding support for Time. It wasn't working for the type that's returned in our app.

**Comment by jscharf** [comment]
Posting here for posterity - this is what it looks like without the coerce changes:
```shell
irb(main):012:0> user = CoerceUser.new
=> #<CoerceUser:0x00007fdade8b91e0>
irb(main):013:0> user.big_decimal
=> 0.12e-1
irb(main):014:0> user.big_decimal.to_f
=> 0.012
```

**Review by jscharf — APPROVED** [review]

**Pull request merged**